pkg/apiserver/event/event.go

```diff
@@ -47,11 +52,52 @@ func NewService(
 
 // Register mounts our HTTP handler on the mux.
 func Register(r *gin.RouterGroup, s *Service) {
-	endpoint := r.Group("/event")
+	endpoint := r.Group("/events")
 
 	// TODO: add more api handlers
-	endpoint.GET("/all", s.listEvents)
+	endpoint.GET("/", s.listEvents)
 }
 
-// TODO: need to be implemented
-func (s *Service) listEvents(c *gin.Context) {}
+// @Summary Get all events from db.
+// @Description Get all chaos events from db.
+// @Tags events
+// @Produce json
+// @Success 200 {array} core.Event
+// @Router /api/events/all [get]
+// @Failure 500 {object} utils.APIError
+func (s *Service) listEvents(c *gin.Context) {
+	name := c.Query("name")
+	namespace := c.Query("namespace")
+	eventList := make([]*core.Event, 0)
```

Comment:
I think the alert is for this variable. Could you please update it to `var eventList []*core.Event`? @fewdan

Reply:
![image](https://user-images.githubusercontent.com/25150124/82674714-bac3e980-9c11-11ea-80c3-41723954fabb.png)

Reply:
Or we don't need to use the variable `resList`, you can just use 

```
var eventList []*core.Event
var err error
eventList, err = s.event.List()
```